torchci/lib/clickhouse.ts

```diff
@@ -0,0 +1,49 @@
+import { createClient } from "@clickhouse/client";
+import { readFileSync } from "fs";
+
+export function getClickhouseClient() {
+  return createClient({
+    host: process.env.CLICKHOUSE_HUD_USER_URL ?? "http://localhost:8123",
```

Comment:
Nit: Please add these new env variables into `torchci/.env.example`, so that other devs know that they are needed to test thing locally